eoflow/models/segmentation_base.py

```diff
@@ -29,7 +31,8 @@
     'accuracy': lambda: tf.keras.metrics.CategoricalAccuracy(name='accuracy'),
     'iou': lambda: MeanIoU(default_max_classes=32),
     'precision': tf.keras.metrics.Precision,
-    'recall': tf.keras.metrics.Recall
+    'recall': tf.keras.metrics.Recall,
+    'mcc': lambda: MCCMetric()
```

Comment:
Isn't this just 
...
`'mcc': MCCMetric`
instead of 
`'mcc': lambda: MCCMetric()  `

Reply:
yes, this should do as well, changed